rust/src/external/ntp.rs

```diff
@@ -0,0 +1,396 @@
+/* Copyright (C) 2017 Open Information Security Foundation
```

Comment:
It can be moved to src/ntp/ntp.rs. The reason I used external is to be able to include all external parsers using one statement in `src/lib.rs`:
```rust
#[cfg(feature = "external")]
pub mod external;
```

Using directories under `src` is no big deal, it will only add one line per module:
```rust
#[cfg(feature = "external")]
pub mod ntp;
#[cfg(feature = "external")]
pub mod ikev2;
```